src/pudl/workspace/datastore.py

```diff
@@ -534,15 +534,15 @@ def _get_pudl_in(args: dict) -> Path:
     if args.pudl_in:
         return Path(args.pudl_in)
     else:
-        return Path(pudl.workspace.setup.get_defaults()["pudl_in"])
+        return Path(pudl.workspace.setup.get_defaults()["PUDL_INPUT"])
 
 
 def _create_datastore(args: argparse.Namespace) -> Datastore:
     """Constructs datastore instance."""
     # Configure how we want to obtain raw input data:
     ds_kwargs = dict(gcs_cache_path=args.gcs_cache_path, sandbox=args.sandbox)
     if not args.bypass_local_cache:
-        ds_kwargs["local_cache_path"] = _get_pudl_in(args) / "data"
+        ds_kwargs["local_cache_path"] = _get_pudl_in(args)
```

Comment:
It would be nice if we could simply replace this with `get_defaults()["data_dir"]`